### PR TITLE
Project setup

### LINKED ISSUE #13: Setup the app
- [x] Setup back
- [x] Setup front
- [x] Setup dev pipelines

### PR DESCRIPTION
**Pull request by Tomansion**
Backend is setup

Frontend is setup

The project is setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+MuseumGo uses ArangoDB as a database, you can install it locally or use a docker image.
+
+### Backend
+
+The backend is made with Python and requests / flask, it is based on a OpenAPI specification, check the [OpenAPI specification](./back/api.yaml) for more information.
+
+#### Requirements
+
+- Python 3.8
+- pip
```
Comment (altristid) [inline README.md]: Which version of pip ?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+- A running ArangoDB instance
```
Comment (altristid) [inline README.md:29]: Which version of ArangoDB ?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+### Frontend
+
+The frontend is made with VueJS.
+
+#### Requirements
+
+- NodeJS v19.0.0c
+- npm
```
Comment (altristid) [inline README.md]: Which version of npm ?

**Review by altristid — APPROVED** [review]

**Review by travisstephane — APPROVED** [review]
Everything is good

**Review by altristid — APPROVED** [review]

**Review by Smartycrouton — APPROVED** [review]
RAS

**Pull request merged**